Makefile

```diff
@@ -28,6 +28,7 @@ all:
 	@echo
 	@echo "Development related targets:"
 	@echo "check:      Runs tree static check, unittests and functional tests"
+	@echo "check-long: Runs tree static check, unittests and long functional tests"
```

Comment:
well it runs both, long and normal functional tests, but I get the point :smile: